src/AddTodoForm.jsx

```diff
@@ -1,16 +1,28 @@
+import { useState } from "react";
+
 export default function AddTodoForm({ onAddTodo }) {
+  const [todoTitle, setTodoTitle] = useState("");
+
+  const handleTitleChange = (event) => {
+    const newTodoTitle = event.target.value;
+    setTodoTitle(newTodoTitle);
```

Comment:
Since newTodoTitle is not used in multiple place,You can shorten the function to one line like this:
```
const handleTitleChange = (event) => setTodoTitle(event.target.value);
```